Routing/Loader/Reader/RestActionReader.php

```diff
@@ -508,6 +518,18 @@ private function getMethodArguments(\ReflectionMethod $method)
                 }
             }
 
+            // Dont break php5 compatibility
+            if (method_exists($argument, 'getType')) {
+                $argumentType = $argument->getType();
+                if ($argumentType) {
+                    foreach ($this->ignoreTypes as $type) {
+                        if ($argumentType === $type) {
```

Comment:
that's broken. The return value of `$argument->getType()` will be a ReflectionType object, so there is no way to have the same instance in the ignoreTypes property.